contrib/pzstd/Pzstd.cpp

```diff
@@ -41,7 +41,7 @@ size_t pzstdMain(const Options& options, ErrorHolder& errorHolder) {
       return 0;
     }
     std::error_code ec;
-    inputSize = file_size(options.inputFile, ec);
+    inputSize = static_cast<size_t>(file_size(options.inputFile, ec));
```

Comment:
Prefer using 64-bits integer for file sizes,
as even on 32-bits platforms, you may have to handle files > 4 GB.

I'm not sure how it would behave  on a 32-bits system if file size was 4GB+1 bytes.
Would `inputSize` value become `1` ?
That could be a use case interesting to add in the test suite by the way.

Conversion to `size_t` can happen at a later stage, when you need to allocate a buffer for example.